### DISCUSSION TITLE
Scribe with DataDog

### DISCUSSION THREAD

**Discussion by mfelipe** [outr/scribe#364] [Q&A] [answered]
Hi there! I'm trying to use Scribe with DataDog but I can't really find any examples. Can anyone give some directions? Thanks!

**Comment by darkfrog26** [comment, marked as answer]
@mfelipe there are a few ways you can go about it.
1. You can write to a standardized log format and have a DD agent read the log files. Personally, I'm not a big fan of this approach, but it seems to be the norm.
2. You can use `scribe-json` to write to JSON-formatted log files (like #1) and use the DD agent to read the log files. This is a bit better as it allows you to use structured data and seems to work much better. However, it's still running the agent.
3. You can use `scribe-logstash` I believe, and configure the logging endpoint (https://docs.datadoghq.com/logs/log_collection/?tab=host#logging-endpoints) to go to DD. I haven't actually tried this, but in theory it should work smoothly. If you decide to go down this path I'd be happy to assist and make sure everything works well. In fact, I'm not opposed to adding a `scribe-datadog` to help make this easier if we can smooth out the edges.

**Reply by mfelipe** [reply]
Thanks @darkfrog26 , I'll go for the second option, it makes more sense for the context I'm in at the moment. But that scribe-datadog is a really good ideia though!
